lib/msf/util/windows_crypto_helpers.rb

```diff
@@ -79,7 +79,7 @@ def decrypt_secret_data(secret, key)
         j = key[j..j+7].length
       end
     end
-    dec_data_len = decrypted_data[0,4].unpack('<L').first
+    dec_data_len = decrypted_data[0,4].unpack('L<').first
```

Comment:
Invalid Ruby redirectives (i.e. Python's way of specifying endianness) are now raised as an error. There's a separate effort to fix all invalid uses of this, but for now I'm fixing the ones that fail the test suite.